inspirehep/modules/workflows/utils/__init__.py

```diff
@@ -158,6 +158,35 @@ def _try_to_log(logfn, *args, **kwargs):
     return _decorator
 
 
+def do_not_repeat(step_id):
+    """Decorator used to skip workflow steps when a workflow is re-run.
+
+    Will store the result of running the workflow step in source_data.persistent_data
+    after running the first time, and skip the step on the following runs.
```

Comment:
maybe add a word about the protocol that the wrapped function should implement to have its writes persisted?